### DISCUSSION TITLE
Automatically appending CPPFLAGS, LDFLAGS, etc

### DISCUSSION THREAD

**Discussion by tinjaw** [Homebrew/discussions#1774] [Everyday usage]
Here is an example of a recent install:   https://pastebin.com/Xj0eQsrD

there are warnings about needing to make modifications to the environment.

1) I am worried that I may have missed some of them in the past and want to know if there is a way to see what those were.
2) Is there a way to get those changes to happed automatically?

**Comment by carlocab** [comment]
If you missed them, you'd know -- something you're using them for wouldn't be working properly. You only need to set things like `LDFLAGS`, `CPPFLAGS`, or `PKG_CONFIG_PATH` if you know what those environment variables do. If you don't, then I recommend against setting them, because you almost certainly don't need them.

As for getting those changes automatically, no, not currently.

To check for more caveats you might have missed, you can do `brew info $(brew list --formula)`.

**Reply by tinjaw** [reply]
I am somewhere in-between as a developer who uses open source libraries. Your last tip there will get me what I need. ty
